internal/kubernetes/operator/project/project.go

```diff
@@ -835,6 +800,75 @@ func buildAlertConfigurations(acProvider store.AlertConfigurationLister, project
 	return results, secretResults, nil
 }
 
+func convertMatchers(atlasMatcher []map[string]interface{}) []akov2.Matcher {
+	res := make([]akov2.Matcher, 0, len(atlasMatcher))
+	for _, m := range atlasMatcher {
+		if matcher, err := toMatcher(m); err == nil {
```

Comment:
please add a comment why this error is ignored, it is confusing and non-idiomatic to ignore errors in Go.

Reply:
ok

Reply:
will instead log a warning.